static/panes/editor.js

```diff
@@ -1411,7 +1408,7 @@ Editor.prototype.onCompileResponse = function (compilerId, compiler, result) {
                         if (colEnd === obj.tag.column) colEnd = -1;
                     }
                 }
-                let link;
+                var link;
```

Comment:
Slowly, slowly, we get better :) Yay!

Reply:
This is actually a backwards step from let to var 🥲
Again, es6 code where it's not meant to be